packages/compile-solidity/test/test_supplier.js

```diff
@@ -150,13 +150,10 @@ describe("CompilerSupplier", function () {
         Config.getTruffleDataDirectory(),
         "compilers/node_modules"
       );
-      const expectedCache = path.resolve(
-        compilerCacheDirectory,
-        "soljson-v0.4.21+commit.dfe3193c.js"
-      );
 
       // Delete if it's already there.
-      if (await fse.exists(expectedCache)) await fse.unlink(expectedCache);
+      if (await fse.exists(compilerCacheDirectory))
+        await fse.removeSync(compilerCacheDirectory);
```

Comment:
Style note: You should probably put braces around the body of this block, indented blocks without braces can be a source of confusion for later editors.

Also, since you're using `removeSync`, the `await` is redundant.